test/runtests.jl

```diff
@@ -342,27 +369,59 @@ const prob_pds_linmod_nonconservative_inplace = PDSProblem(linmodP!, linmodD!, [
                               dt, adaptive = false)
             @test sol_MPE_ip.t ≈ sol_MPE_ip_2.t ≈ sol_IE_ip.t
             @test sol_MPE_ip.u ≈ sol_MPE_ip_2.u ≈ sol_IE_ip.u
-
-            # Check different linear solvers
-            dt = 0.25
-            sol1 = solve(prob_ip, MPE(); dt)
-            sol1_2 = solve(prob_ip_2, MPE(); dt)
-            sol2 = solve(prob_ip, MPE(linsolve = RFLUFactorization()); dt)
-            sol2_2 = solve(prob_ip_2, MPE(linsolve = RFLUFactorization()); dt)
-            sol3 = solve(prob_ip, MPE(linsolve = LUFactorization()); dt)
-            sol3_2 = solve(prob_ip_2, MPE(linsolve = LUFactorization()); dt)
-            @test sol1.t ≈ sol2.t ≈ sol3.t ≈ sol1_2.t ≈ sol2_2.t ≈ sol3_2.t
-            @test sol1.u ≈ sol2.u ≈ sol3.u ≈ sol1_2.u ≈ sol2_2.u ≈ sol3_2.u
         end
 
-        @testset "Linear model problem (nonconservative)" begin
-            dt = 0.25
-            sol_MPE_op = solve(prob_pds_linmod_nonconservative, MPE(); dt)
-            sol_MPE_ip = solve(prob_pds_linmod_nonconservative_inplace, MPE(); dt)
-            @test sol_MPE_ip.t ≈ sol_MPE_op.t
-            @test sol_MPE_ip.u ≈ sol_MPE_op.u
+        # Here we check that different linear solvers can be used
+        #TODO: Add other 3rd order schemes
+        @testset "Different linear solvers" begin
+            # problem data
+            u0 = [0.9, 0.1]
+            tspan = (0.0, 2.0)
+            p = [5.0, 1.0]
+
+            # analytic solution
+            function f_analytic(u0, p, t)
+                u₁⁰, u₂⁰ = u0
+                a, b = p
+                c = a + b
+                return ((u₁⁰ + u₂⁰) * [b; a] +
+                        exp(-c * t) * (a * u₁⁰ - b * u₂⁰) * [1; -1]) / c
+            end
+
+            # linear model problem - in-place
+            function linmodP!(P, u, p, t)
+                fill!(P, zero(eltype(P)))
+                P[1, 2] = p[2] * u[2]
+                P[2, 1] = p[1] * u[1]
+                return nothing
+            end
+            function linmodD!(D, u, p, t)
+                fill!(D, zero(eltype(D)))
+                return nothing
+            end
+            prob_ip = PDSProblem(linmodP!, linmodD!, u0, tspan, p; analytic = f_analytic)
+            prob_ip_2 = ConservativePDSProblem(linmodP!, u0, tspan, p;
+                                               analytic = f_analytic)
+
+            algs = (MPE, (; kwargs...) -> MPRK(1.0; kwargs...))
```

Comment:
```suggestion
            algs = (MPE, (; kwargs...) -> MPRK22(1.0; kwargs...))
```

Sorry, my fault

Reply:
Great! I'll add more 2nd and 3rd order schemes to this test.